packages/client/ui/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1 @@
+export {Root as Tooltip} from '@radix-ui/react-tooltip'
```

Comment:
re-exporting the primitive so future devs don't see radix-ui, rather they just import from the /ui directory